daras_ai_v2/bot_integration_add.py

```
@@ -0,0 +1,288 @@
+        gui.html(
+            '<a class=" btn btn-theme btn-secondary w-100 mb-3" '
+            'href="https://t.me/BotFather?text=/newbot" target="_blank">'
+            f"{icons.telegram} Open BotFather"
+            "</a>",
+        )
+        gui.markdown("**Step 2: Name your bot & add a username**")
+        gui.markdown("Send: *`/newbot`* to BotFather to get started")
+        _copiable_field("Suggested bot name", bot_name)
+        _copiable_field("Suggested bot username", bot_username)
+
+        gui.markdown("**Step 3: Paste in your bot’s token**")
```

Ruff (0.15.10)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)